Content.Client/SimpleStation14/Overlays/Systems/SSDIndicatorSystem.cs

```diff
@@ -32,7 +28,7 @@ public override void Update(float time)
 
             var sprite = _entity.GetComponent<SpriteComponent>(uid);
 
-            if (indicator.IsSSD)
+            if (indicator.IsSSD && _config.GetCVar(SimpleStationCCVars.SSDIndicatorEnabled))
```

Comment:
This is gross. Make a comment that you're aware this is gross and it should be cleaned at a later date.
Feel free to just quote this.
Because this is gross.

Reply:
I can't think of any better ways to do it